app/components/TracksCard/index.js

```diff
@@ -4,37 +4,209 @@
+        height: 100%;
+
+        div.ant-image-img {
+          height: 100%;
+          border-radius: 0.5rem;
+        }
+
+        div.ant-image-mask {
+          border-radius: 0.5rem;
+        }
+      }
+
+      img {
+        height: 100%;
+        border-radius: 0.5rem;
+      }
+    }
+
+    div.ant-card-body {
+      padding: 0.75rem 0.25rem 0.25rem;
+      height: 50%;
+      display: grid;
+      grid-template-rows: 0.5fr 1fr 2fr 1fr;
+      gap: 0.5rem;
+
+      &::after,
+      &::before {
+        display: none;
+        content: none;
+      }
+    }
   }
 `;
 
-const CustomText = styled.p`
-  margin: 10px;
-  font-size: 1rem;
+const CardHeader = styled.p`
+  font-size: 1.2rem;
+  font-weight: 700;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+    'Helvetica Neue', sans-serif;
+  color: ${colors.text};
+  letter-spacing: 0.1px;
+  padding-left: 0.25rem;
+  line-height: 1.2;
+  display: block; /* Fallback for non-webkit */
+  display: -webkit-box;
+  -webkit-line-clamp: 1; /* no of lines */
+  text-overflow: ellipsis;
+  overflow: hidden !important;
+  -webkit-box-orient: vertical;
+`;
+
+const CardDescription = styled.span`
+  font-size: 0.75rem;
+  font-weight: 400;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+    'Helvetica Neue', sans-serif;
+  opacity: 0.9;
+  padding-left: 0.25rem;
+  color: ${colors.text};
+  display: block; /* Fallback for non-webkit */
+  display: -webkit-box;
+  -webkit-line-clamp: 3; /* no of lines */
+  line-height: 1.4;
+  text-overflow: ellipsis;
+  overflow: hidden !important;
+  -webkit-box-orient: vertical;
 `;
 
-export function TracksCard({ collectionName, trackName, artworkUrl100: imgUrl /*previewUrl*/ }) {
+const PlayBtn = styled(Button)`
+  && {
+    background-color: ${colors.primaryLight};
+    border: none;
+    color: ${colors.secondary};
+    border-radius: 0.5rem;
+    font-size: 0.75rem;
+    font-weight: 500;
+    width: 100%;
+
+    &:hover {
+      background-color: ${colors.primaryDark};
+    }
+  }
+`;
+
+const BtnContainer = styled.div`
+  display: flex;
+  gap: 0.75rem;
+`;
+
+const MoreBtn = styled(Button)`
+  && {
+    background-color: ${colors.text};
+    border: none;
+    color: ${colors.secondary};
+    border-radius: 0.5rem;
+    font-size: 0.75rem;
+    font-weight: 500;
+    width: 100%;
+
+    &:hover {
+      opacity: 0.8;
+    }
+  }
+`;
+
+const ButtonText = styled.span`
+  font-size: 0.75rem;
+  font-weight: 500;
+`;
+
+const CustomTags = styled.span`
+  background-color: ${colors.text};
+  opacity: 0.8;
+  width: fit-content;
+  padding: 0.2rem 0.5rem;
+  height: 1.5rem;
+  font-size: 0.75rem;
+  border-radius: 2rem;
+  font-weight: 500;
+`;
+
+// --------TracksCard Component--------------
+
+export function TracksCard({
+  collectionName,
+  trackName,
+  artworkUrl100: imgUrl,
+  previewUrl,
+  handleGlobalClick,
+  kind: tags,
```

Comment:
we can just keep it is tags and use the same name while passing them as well

Reply:
I will correct those